### ISSUE TITLE
<!-- routify:options bundle=true --> make app error

### ISSUE THREAD

**Issue by oschina** [sveltech/svelte-filerouter#425]
**bug description**
when put `<!-- routify:options bundle=true -->` in `pages/_layout.svelte` , it build ok, but error happend when visit it. 

if remove this, it works.

Error:
```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'name')
    at $e (vendor.f4667ffd.js:4:9323)
    at Object.Un.e.$$.update (vendor.f4667ffd.js:4:23245)
    at an (vendor.f4667ffd.js:4:2556)
    at je (vendor.f4667ffd.js:4:2308)
``` 
vendor.f4667ffd.js:4:9323
```
s=e.componentFile.name.replace(/Proxy<_?(.+)>/,"$1").replace(/^Index$/,e.component.shortPath.split("/").pop())
```

**vite.config.js**
```
import { defineConfig } from 'vite';
import { svelte } from '@sveltejs/vite-plugin-svelte';
import { resolve }  from 'path';

const production = process.env.NODE_ENV === 'production'

export default defineConfig({
    server: {
        port: 5000
    },

    plugins: [
        svelte({
            emitCss: true,
            compilerOptions: {
                dev: !production,
            },
        })
    ],

    resolve: {
        alias: {
            $api : resolve('./src/api'),
            $components : resolve("./src/components")
        }
    }
});
```

**version**
Routify 2.18.4, Svelte 3.46.1, vite 2.7.12

**screenshot**

![image](https://user-images.githubusercontent.com/1540855/149658197-b7803df4-f878-4c33-8311-e9e207f7f472.png)

![image](https://user-images.githubusercontent.com/1540855/149667857-3ceb0369-90fa-44d8-a3e2-da45f1479d73.png)

**Comment by oschina** [comment]
bundle="true" ！！！

**Comment by oschina** [comment]
I download `https://github.com/roxiness/routify-starter` and add `<!-- routify:options bundle=true -->` to `pages/_layout.svelte`

the same error occured.

**Comment by jakobrosenberg** [comment]
Hi. Would you be able to provide a minimal reproducible for this?

**Comment by oschina** [comment]
> Hi. Would you be able to provide a minimal reproducible for this?

I download https://github.com/roxiness/routify-starter and add `<!-- routify:options bundle=true -->` to pages/_layout.svelte

build & preview,  the same error occured.

**Comment by oschina** [comment]
https://github.com/oschina/routify-starter
Here is my demo project , only modify src/public/_layout.svelte to `<!-- routify:options bundle=true -->`

npm install
npm run build
npm run preview

open `http://localhost:5000` then error happened

**Comment by LexShade** [comment]
Same problem here with the routify-starter.

When i execute

npm run dev

I get the following error:

Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'name')
    at suppressComponentWarnings (index.js:66:34)
    at Object.$$self.$$.update (Route.svelte? [sm]:106:18)
    at update (index.mjs:1071:12)
    at flush (index.mjs:1042:13)

[localhost-1646323553978.log](https://github.com/roxiness/routify/files/8179228/localhost-1646323553978.log)

**Comment by aakash14goplani** [comment]
@jakobrosenberg @ghostdevv
Any updates on this? I am facing similar issue!

**Comment by theetrain** [comment]
I am facing this as well, and removing `<!-- routify:options bundle=true -->` allows the site to run.

Environment:
- @roxi/routify@2.18.8
- vite@4.0.4
- npm@8.19.2
- node@18.12.1

**Comment by jakobrosenberg** [comment]
This should be fixed in 2.18.9. Sorry about the delay.

**Comment by theetrain** [comment]
It works now, I appreciate it!
